src/sql/workbench/services/notebook/browser/notebookServiceImpl.ts

```diff
@@ -248,30 +248,60 @@ export class NotebookService extends Disposable implements INotebookService {
 		lifecycleService.onWillShutdown(() => this.shutdown());
 	}
 
-	public async createNotebookInput(options: INotebookShowOptions, resource?: UriComponents): Promise<IEditorInput | undefined> {
+	private getUntitledFileUri(): URI {
```

Comment:
This seems like something that could go into a generic helper module instead (and get passed in the prefix to use)